play-ws-standalone/src/main/scala/play/api/libs/ws/StandaloneWSRequest.scala

```diff
@@ -138,7 +146,17 @@ trait StandaloneWSRequest {
     val newHeaders = headers.toList.flatMap { param =>
       param._2.map(p => param._1 -> p)
     } ++ hdrs
-    withHttpHeaders(newHeaders: _*)
+
+    // Content-Type is single valued:
+    // https://tools.ietf.org/html/rfc7231#section-3.1.1.5
+    //
+    // So we should undo the append made above.
+    hdrs.find(_._1.equalsIgnoreCase("Content-Type")) match {
```

Comment:
Can't be done without touching the subproject dependencies. The trait `StandaloneWSRequest` is part of `play-ws-standalone` which does not depends on the shaded AHC (`HttpHeaders.Names` is from AHC). :-(